Source/CMakeLists.txt

```diff
@@ -211,9 +211,11 @@ IF (NOT (${CMAKE_VERSION} VERSION_LESS 3.4))
   # Configure WebSocket
   # Create the link to the WebSocket libraries
   IF(OpenIGTLink_USE_WEBSOCKET)
-    FIND_PACKAGE(WebSocket REQUIRED)
-    LIST(APPEND LINK_LIBS WebSocket)
-
+    FIND_PACKAGE(websocketpp REQUIRED)
+    INCLUDE(${CMAKE_CURRENT_SOURCE_DIR}/WebSocket/UseWebSocket.cmake)
```

Comment:
Ideally this file should be defined as a variable in websocketppConfig.cmake

Reply:
websocketppConfig.cmake is managed by another open source project, let me try if i am able to  pull a request there